gloo/docs/enterprise/rate_limits_envoy/_index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Envoy Rate Limits
+weight: 3
+---
+
+In this document we will show how to use Gloo with Rate limits.
+
+Gloo enterprise comes with a rate limit server based off of lyft's rate-limit server.
```

Comment:
lyft -> Lyft

on second thought, do you think it's valuable to say based on Lyft's ratelimit server? I think this is implementation detail and not necessarily helping our user. not opposed to it, just wondering what value it adds

Reply:
i think it will help the user as they document its behavior